lib/insights.ts

```diff
@@ -54,9 +54,9 @@ class AlgoliaAnalytics {
   _apiKey: string;
   _appId: string;
   _region: string;
-  _endpointOrigin: string;
+  _endpointOrigin = "https://insights.algolia.io";
   _userToken: string;
-  _userHasOptedOut: boolean;
+  _userHasOptedOut = false;
   _useCookie: boolean;
   _cookieDuration: number;
```

Comment:
Don't we wnat to set defaults for those two as well?

Reply:
I think we can leave them as `undefined` by default since `_useCookie` is only used in `init()` and `_cookieDuration` only used in `setAnonymousUserToken()` which is used in `init()`, WDYT?

Reply:
My point is, if we say that we no longer need to `init` to use Insights, these defaults need to be defined no? `init` just becomes a way to change those options, but they need to be defaulted somewhere.

Reply:
It makes sense, thanks @sarahdayan!